artemis-core/src/main/java/org/jnosql/artemis/DatabaseType.java

```diff
@@ -24,5 +24,18 @@
  * Defines the database type supported on Artemis.
  */
 public enum DatabaseType {
-    DOCUMENT, COLUMN, KEY_VALUE, GRAPH;
+    DOCUMENT("DocumentCollectionManager"),
```

Comment:
Why do we need this description eg: `DocumentCollectionManager` ?
Once we work also with async manager should we have a new type?
IMHO: I would remove it.